app/javascript/entrypoints/admin.tsx

```diff
@@ -285,6 +285,51 @@ async function mountReactComponent(element: Element) {
   );
 }
 
+// In the account warning interface, email notifications imply in-app notifications, so update accordingly
+Rails.delegate(
```

Comment:
More of a question than a feedback, but -- if we are planning on migrating away from rails-ujs over time, this creates slightly more work. If there's an easy/alternate way to set up the hooks, maybe do that? If not, fine as-is and we can migrate whenever other areas migrate (or never).